### PR TITLE
[ISSUE #3118]Refactor Acl

### LINKED ISSUE #3118: [Enhancement] Refactor Acl
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

Refactor Acl 

### Describe the solution you'd like

Refactor Acl 

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by mxsm**
Fixes #3118 

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Refactor Acl*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java [hunk 1]
```diff
@@ -26,14 +26,25 @@
...
 
 import java.util.Properties;
-import java.util.ServiceLoader;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class Acl {
+
     private static final Logger logger = LoggerFactory.getLogger(Acl.class);
```
Comment (jonyangx) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:35]: Simplify code using Lombok @Slf4j.

Comment (mxsm) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:35]: @jonyangx  I want to submit a separate PR to unified fix the logging issue. Use  lombok @slf4j annotation consistently

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java [hunk 2]
```diff
@@ -26,14 +26,25 @@
...
+
+    private static final Acl INSTANCE = new Acl();
+
+    private AclService aclService;
+
+    private Acl() {
+
+    }
+
+    public static Acl getInstance() {
```
Comment (jonyangx) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:45]: Suggest merge getInstance() and init() method.
eg. public static Acl getInstance(String aclPluginType) .
Add collection for Acl instannce and pluginType mapping.

Comment (mxsm) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:45]: @jonyangx  Thanks for you suggest. I will do it and resubmit later!

Comment (jonyangx) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:45]: @mxsm Pls Consider start() and stop()  methods, how to manage thd aclService lifecycle.

Comment (mxsm) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:45]: @jonyangx  OK, I will

**Comment by mxsm** [comment]
@jonyangx  The lifetime of the ACL is the same as the lifetime of the EventMeshServer. PTAL~ thanks

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java [hunk 3]
```diff
@@ -25,33 +25,44 @@
...
             throw new RuntimeException("doesn't load the aclService plugin, please check.");
         }
+        Acl acl = new Acl();
+        acl.aclService = aclServiceExt;
+
+        return acl;
+    }
+
+    public void init() throws AclException {
         aclService.init();
```
Comment (jonyangx) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:66]: Suggest add init,start,stop flag to avoid call multiple times. @mxsm

Comment (mxsm) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java:76]: @jonyangx  done

**Review by jonyangx — APPROVED** [review]

**Pull request merged**